backends/blocked/ceed-blocked-operator.c

```diff
@@ -282,29 +322,46 @@ static int CeedOperatorApply_Blocked(CeedOperator op, CeedVector invec,
       CeedChk(ierr);
       ierr = CeedQFunctionFieldGetNumComponents(qfinputfields[i], &ncomp);
       CeedChk(ierr);
+      // Restrict active input block
```

Comment:
For now I am applying this only to the active input - I am concerned that applying this to all inputs will decrease performance by extra memory movement for unchanged input vectors